docs/docs/cmd/yammer/yammer-search.md

```diff
@@ -0,0 +1,77 @@
+# yammer search
+
+Returns a list of messages, users, topics and groups that match the user’s search query.
+
+## Usage
+
+```sh
+m365 yammer search [options]
+```
+
+## Options
+
+`-h, --help`
+: output usage information
+
+`-s, --search <search>`
+: The query for the search
+
+`--show <show>`
+: Specifies the type of data to return when using --ouptut text. Allowed values Summary|Messages|Users|Topics|Groups.
+
+`--limit [limit]`
+: Limits the results returned for each item category.
+
+`-o, --output [output]`
+: Output type. `json,text`. Default `text`
+
+`--verbose`
+: Runs command with verbose logging
+
+`--debug`
+: Runs command with debug logging
+
+## Remarks
+
+!!! attention
+    In order to use this command, you need to grant the Azure AD application used by the CLI for Microsoft 365 the permission to the Yammer API. To do this, execute the `cli consent --service yammer` command.
+
+    The command without `--output json` will just return the search summary for the query.
+
+## Examples
+
+Returns the search result for the query `community`
+
+```sh
+m365 yammer search --search "community"
+```
+
+Returns the groups found with the search query `community`
+
+```sh
+m365 yammer search --search "community" --show "groups"
```

Comment:
We use lowercase `groups` here, but in the allowed values use `Groups` with capital `G`. Let's choose one format and use it consistently

Reply:
Changed everything to lowercase